### ISSUE TITLE
Programs to use on Linux (not an issue)

### ISSUE THREAD

**Issue by Minskal** [xerpi/vita-udcd-uvc#33]
Hi,
I'm a Linux user, specifically Slackware 14.2.
I successfully installed the plugin you implemented, but had problems making it work with programs like vlc, mvp etc... on Linux.
Then I tried a program called guvcview, and managed to make it work flawlessly.
I wrote this issue, even it is not an issue, because I thought I could share my experience in case others had problems setting it up like me.

The interface is really simple and user-friendly, but I'll attach an image with the correct settings needed to properly display the ps vita screen.
First of all, plug in the ps vita (with the plugin already installed, of course), then open guvcview and select the video device to "ps vita" (it's auto-detected thanks to the plugin). After this set the video options like the image, and that's all.

Thanks again for this plugin.

![image](https://user-images.githubusercontent.com/19166529/63635058-5235e500-c65e-11e9-8264-5eb978c783f9.png)

**Comment by xerpi** [comment]
Hi, did you select the proper /dev/videoX in mpv/mplayer?

**Comment by Minskal** [comment]
Yes, (in my case /dev/video1 or /dev/video2) but either it fails or a green screen appears. In any case I get satisfying results using guvcview.

**Comment by RexNihil** [comment]
Thank you so much, I've tried everything, and gucview worked perfectly with no setting changes, soon as I plugged in the USB it opened the video with no tearing or color issues or weird setup necessary. I would suggest that this be added in the Linux section instead of mplayer or mpv, neither of which I could get to play the stream properly.
